frontend/__tests__/unit/components/BarChart.test.tsx

```diff
@@ -0,0 +1,130 @@
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faFire } from '@fortawesome/free-solid-svg-icons'
+import { act, render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import { ThemeProvider } from 'next-themes'
+import React from 'react'
+
+import BarChart from 'components/BarChart'
+
+// Register FontAwesome icon
```

Comment:
I believe there are better ways to test this 🤔 Please check how other test files check for rendered icons.